.github/workflows/ci_compatible.yml

```diff
@@ -112,8 +112,16 @@ jobs:
   watcher:
     runs-on: ubuntu-latest
     needs: pytester
+    if: always()
     steps:
     - run: echo "${{ needs.pytester.result }}"
+    - name: failing...
+      if: needs.pytester.result == 'failure'
+      run: exit 1
+    - name: cancelled or skipped...
+      if: contains(fromJSON('["cancelled", "skipped"]'), needs.pytester.result)
+      timeout-minutes: 1
+      run: sleep 90
```

Comment:
This will always fail because it sleeps 90 seconds and the timeout for the step is 60 seconds, right?

Reply:
yes, the output shall be canceled and the tester state